src/hooks/useVersionHistory/getActionLabel.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Gets translated change type label
+ * @param {function} formatMessage
+ * @returns {{ADDED, MODIFIED, REMOVED}}
+ */
+export const getActionLabel = formatMessage => {
+  return {
+    ADDED: formatMessage({ id: 'stripes-acq-components.audit-log.action.added' }),
+    MODIFIED: formatMessage({ id: 'stripes-acq-components.audit-log.action.edited' }),
+    REMOVED: formatMessage({ id: 'stripes-acq-components.audit-log.action.removed' }),
```

Comment:
oops, ids prefixes should be changed to `ui-inventory`, there are no such translations in stripes-acq-components anymore